lambdas/commands.py

```diff
@@ -6,7 +6,11 @@
 
 
 def generate_extract_command(
-    input_data: dict, run_date: str, timdex_bucket: str, verbose: bool
+    # ruff: noqa: FBT001
```

Comment:
Which error code?  The [ruff linting rule `FBT001`](https://docs.astral.sh/ruff/rules/boolean-type-hint-positional-argument/)?

Yeah, as we migrate projects, been seeing that.  I can understand the logic behind not using positional boolean arguments in a function, but sometimes it does seem appropriate.  As we ease into these new linting rules, I'm quite happy with this being an explicit skip for the time being when the situation calls for it, but _not_ necessarily at the file or project level, because I do think it helps pause and think if a bool arg is a good idea.